pegasus/sites.v3/code.org/public/teacher-dashboard/section_stats.haml

```diff
@@ -5,6 +5,7 @@ content-type: text/ng-template
 ---
 
 #uitest-stats-tab{style: "padding-left:20px", 'ng-init' => '$emit("stats-table-rendered");'}
+  %div.alert.alert-warning The Completed Level count is temporarily disabled and will return in a few days. In the meantime, you can still access level-by-level student progress on the Progress tab.
```

Comment:
Think we should link to a blog post or something?

Reply:
@jeremydstone or @ryansloan do we need a blog post or support article for this?

Reply:
Just chatted with Ryan, we think this message is sufficient for these teachers.